Updated the note for custom sync rule for AAD Connect (#107681)
Azure/azure-content: articles/active-directory/governance/how-to-lifecycle-workflow-sync-attributes.md

   18. Enable the scheduler again by running `Set-ADSyncScheduler -SyncCycleEnabled $true`.

> [!NOTE]
>- **msDS-cloudExtensionAttribute1** is an example source.

> >- **Starting [Azure AD Connect 2.0.3.0](../hybrid/reference-connect-version-history.md#functional-changes-10), employeeHireDate is added to the default 'Out to AAD' rule so steps 10-16 is not required.**

Reviewer: ```suggestion
>- **Starting with [Azure AD Connect 2.0.3.0](../hybrid/reference-connect-version-history.md#functional-changes-10), `employeeHireDate` is added to the default 'Out to Azure AD' rule, so steps 10-16 are not required.**
```

> >- **Starting [Azure AD Connect 2.0.3.0](https://learn.microsoft.com/azure/active-directory/hybrid/reference-connect-version-history#functional-changes-10), employeeHireDate is added to the default 'Out to AAD' rule so steps 10-16 is not required.**

Reviewer: ```suggestion
>- **Starting [Azure AD Connect 2.0.3.0](../hybrid/reference-connect-version-history.md#functional-changes-10), employeeHireDate is added to the default 'Out to AAD' rule so steps 10-16 is not required.**
```
fixing link
